app/charges/controllers.py

```diff
@@ -69,8 +69,9 @@ def get_charges(user, user_data, broadcast = False):
     membership = committee.members.filter_by(member= user).first()
 
     for charge in charges:
-        if charge.private and membership is None:
-            continue;
+        if charge.private:
+            if (membership is None or membership.role != Roles.CommitteeHead) or not user.is_admin or not user.is_super:
```

Comment:
This could be broken down into two conditionals if we want better readability.

Reply:
Yeah, sorry I just am seeing this now, I think that is the way to go. Always best to have the next guy in mind, the easier to understand the better.